steam/ext/dota2/models.py

```diff
@@ -0,0 +1,197 @@
+    series_id: int
+    teams: list[TournamentTeam]
+    battle_cup_info: BattleCup | None
+
+
+class LiveMatch:
+    """Represents a live match of Dota 2
+
+    Attributes
+    -----------
+    match_id
+        Match ID.
+    server_steam_id
+        Server Steam ID.
+    lobby_id
+        Lobby ID.
+    lobby_type
+        Lobby Type. i.e. Ranked, Unranked.
+    game_mode
+        Game mode, i.e. All Pick, Captains Mode.
+    players
+        List of players in this match. This is sorted with team slot order (or commonly referred as player's colour).
+        Radiant team players first, i.e. "blue", "teal", "purple", ... Then Dire team players.
+    average_mmr
+        Average MMR.
+    sort_score
+        Number for in-game's Watch Tab to sort the game list in descending order with.
+    spectators:
+        Amount of people currently watching this match live in the Dota 2 application.
+    start_time:
+        Datetime in UTC for when the match started, including pre-draft stage.
+    end_time:
+        Datetime in UTC for when the match is finished. 0 if match is currently live.
+    game_time:
+        Timedelta representing in-game timer.
+        Draft stage and time before runes (0:00 mark) have this as non-positive timedelta.
+    delay
+        Time delay for the match if watching in the Dota 2 application.
+        Similarly, the data in attributes of this class is also behind the present by this delay.
+    last_update_time:
+        Time the data was updated last by the Game Coordinator.
+    tournament
+        Tournament information, if the match is a tournament game.
+    radiant_lead
+        Amount of gold lead Radiant team has. Negative value in case Dire is leading.
+    radiant_score
+        Amount of kills Radiant team has
+    dire_score
+        Amount of kills Dire team has
+    building_state
+        Bitmask. An integer that represents a binary of which buildings are still standing.
+    custom_game_difficulty
+        Custom Game Difficulty
+    """
+
+    def __init__(self, state: GCState, proto: CSourceTVGameSmall) -> None:
+        self._state: GCState = state
+
+        self.match_id: int = proto.match_id
+        self.server_steam_id: int = proto.match_id
+        self.lobby_id: int = proto.lobby_id
+
+        self.lobby_type = CSODOTALobbyLobbyType.try_value(proto.lobby_type)
+        self.game_mode = DOTAGameMode.try_value(proto.game_mode)
+        self.average_mmr: int = proto.average_mmr
+        self.sort_score: int = proto.sort_score
+        self.spectators: int = proto.spectators
+
+        self.start_time = datetime.datetime.fromtimestamp(proto.activate_time, datetime.timezone.utc)
+        self.game_time = datetime.timedelta(seconds=proto.game_time)
+        self.end_time = datetime.timedelta(seconds=proto.deactivate_time)
+        self.delay = datetime.timedelta(seconds=proto.delay)
+        self.last_update_time = datetime.datetime.fromtimestamp(proto.last_update_time, datetime.timezone.utc)
```

Comment:
Please use steam.utils.DateTime.from_timestamp here and import timedelta directly from datetime